protection.lua

```diff
@@ -90,14 +90,39 @@ minetest.register_craftitem("simple_protection:claim", {
 minetest.register_alias("simple_protection:claim_stick", "simple_protection:claim")
 minetest.register_alias("claim_stick", "simple_protection:claim")
 
-minetest.register_craft({
-	output = "simple_protection:claim",
-	recipe = {
-		{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
-		{"default:steel_ingot", "default:stonebrick", "default:steel_ingot"},
-		{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
-	}
-})
+-- If neither one of these occurs then the claim stick isn't craftable
+if minetest.registered_items["default:dirt"] then
+	minetest.register_craft({
+		output = "simple_protection:claim",
+		recipe = {
+			{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
+			{"default:steel_ingot", "default:stonebrick", "default:steel_ingot"},
+			{"default:copper_ingot", "default:steel_ingot", "default:copper_ingot"},
+		}
+	})
+elseif minetest.registered_items["mcl_core:dirt"] then
+	if minetest.registered_items["mcl_copper:stone_with_copper"] then
```

Comment:
This check is fine (the material is actually related), but consider using variables like demonstrated in pipeworks.

Reply:
Yeah didn't know how high tech you wanted me to make this mod. (I'll redo it with as best I know)

Reply:
I see that it takes some effort to get it there, but afterwards it will be easy to extend to support more games.

Reply:
Yup, just need to work on getting it to that spot in the most reliable manner.